saleor/core/taxes.py

```diff
@@ -82,3 +91,125 @@ class TaxData:
     shipping_price_net_amount: Decimal
     shipping_tax_rate: Decimal
     lines: List[TaxLineData]
+
+
+WEBHOOK_TAX_CODES_CACHE_KEY = "webhook_tax_codes"
+
+DEFAULT_TAX_CODE = "O9999999"
+DEFAULT_TAX_DESCRIPTION = "Unmapped Other SKU - taxable default"
+
+
+def _get_current_tax_app() -> App:
+    """Return currently used tax app or None, if there aren't any."""
```

Comment:
If we can return `None` I think we should change the function signature to `-> Optional[App]`

Reply:
Resolved in recent commit.